applications/java/functest/javaFunctest.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+echo "This script is performing the Funnctional test for Java application"
+
+echo "Based on the input parameter, color marked will be deemded as functionality failure"
```

Comment:
this sentence is confusing. let's change to  - `The expected color is defined by the input parameter. Failure: if the color output by the app is different than the expected"

Reply:
done